Library/Formula/sourcekitten.rb

```diff
@@ -0,0 +1,16 @@
+class SourceKitten < Formula
```

Comment:
SourceKitten is the framework, `sourcekitten` is the command line tool. This really just installs the command line tool.